client/components/domains/search-filters/tld-filter-bar.jsx

```diff
@@ -99,10 +101,23 @@ export class TldFilterBar extends Component {
 			return this.renderPlaceholder();
 		}
 
-		const className = classNames( 'search-filters__buttons', {
+		const className = classNames( '', {
+			'search-filters__buttons': ! this.props.showDesignUpdate,
+			'search-filters__checkboxes': this.props.showDesignUpdate,
 			'search-filters__tld-filter-bar--is-domain-management': ! this.props.isSignupStep,
 		} );
 
+		if ( this.props.showDesignUpdate ) {
+			return (
+				<CompactCard className={ className }>
+					<FormLegend className="search-filters__filter-by">Filter by: </FormLegend>
```

Comment:
I know you're probably testing English only, but if you want to save yourself one nanosecond of work later, this string is translated! :)

https://translate.wordpress.com/projects/wpcom/-all-translated/25225/

Reply:
Thanks, implemented in [517527c](https://github.com/Automattic/wp-calypso/pull/39346/commits/517527c44eb4b7dd43fbf995cfa270113dc23db1).